src/main/java/edu/greenblitz/robotName/Robot.java

```diff
@@ -49,13 +50,14 @@ public enum RobotType {
 
 	@Override
 	public void robotInit() {
-		Pathfinding.setPathfinder(new LocalADStar());
+//		Pathfinding.setPathfinder(new LocalADStar());
 		CommandScheduler.getInstance().enable();
-		initializeLogger();
-		initializeAutonomousBuilder();
-		initializeSubsystems();
-		SwerveChassis.getInstance().resetAllEncoders();
-		OI.init();
+//		initializeLogger();
+//		initializeAutonomousBuilder();
+//		initializeSubsystems();
+//		SwerveChassis.getInstance().resetAllEncoders();
+		LED.init();
```

Comment:
put the `LED.init()` in the `initializeSubsystems`

Reply:
yoav - i am very agriring with this!!!!